src/style.css

```diff
@@ -1,3 +1,56 @@
-body {
+html {
+  font-family: Roboto, serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+body,
+nav,
+ul {
```

Comment:
Not good to add `nav` and `ul` here.
If what you are trying to accomplish here is to reset the styling for everything, which is ok, you could use `*`, just for this purpose.
If the styling is meant only for the `nav` and `ul`, you should add these in their classes, not in the tag itself